proof/refine/ARM/Ipc_R.thy

```diff
@@ -2860,12 +2829,20 @@ lemma schedContextDonate_invs':
    apply (rule_tac hoare_weaken_pre[OF hoare_pre_cont])
    apply (clarsimp simp: obj_at'_def sym_refs_asrt_def)
   apply (wp schedContextDonate_valid_pspace' schedContextDonate_vrq
-                    schedContextDonate_valid_queues schedContextDonate_valid_queues'
-                    schedContextDonate_valid_idle' schedContextDonate_if_live_then_nonz_cap')
+            schedContextDonate_valid_queues schedContextDonate_valid_queues'
```

Comment:
It seems unusual that so many of these were added in, is there an innocent reason for this?

Reply:
Was just temporary, it's removed now